ringpop.go

```diff
@@ -188,15 +188,15 @@ func (rp *Ringpop) identity() (string, error) {
 // r.channelIdentityResolver resolves the hostport identity from the current
 // TChannel object on the Ringpop instance.
 func (rp *Ringpop) channelIdentityResolver() (string, error) {
-	hostport := rp.channel.PeerInfo().HostPort
+	peerInfo := rp.channel.PeerInfo()
 	// Check that TChannel is listening. By default, TChannel listens on an
 	// ephemeral host/port. The real port is then assigned by the OS when
 	// ListenAndServe is called. If the hostport is 0.0.0.0:0, it means
```

Comment:
Comment-nitpick: looking at `connection.go:53`: it should say "ends with `:0`" instead (this can happen if service is listening, e.g., on loopback address, and specifying the zero port). Code is fine though.